stdlib/test/csv/people.csv

```diff
@@ -0,0 +1,3 @@
+Name,Age,Gender
+"Peter Smith",23,Male
```

Comment:
```suggestion
"Peter Smith",23,'Male'
```

(wrt to previous comment) 

With `quotechar='"'` apart of your tests, might be beneficial to test output as `["Peter Smith", "23", "'Male'"]`

Reply:
Good idea! Done: https://github.com/msaelices/mojo/commit/9f620b25d5e549d89d9c8de396d018ac1f12c1d7